trace/trace.go

```diff
@@ -350,6 +350,9 @@ type Span interface {
 	// AddEvent adds an event with the provided name and options.
 	AddEvent(name string, options ...EventOption)
 
+	// AddLink adds link after span starts.
```

Comment:
This needs an additional disclaimer required by the specification
```suggestion
	// AddLink adds link after span starts.
	// Adding links at span creation using WithLinks is preferred to calling AddLink
	// later, for contexts that are available during span creation, because head
	// sampling decisions can only consider information present during span creation.
```